tasks/process_agent.py

```diff
@@ -32,7 +22,6 @@ def build(
     build_include=None,
     build_exclude=None,
     flavor=AgentFlavor.base.name,
-    go_version=None,
```

Comment:
Just for my own understanding, why is this being removed?

Reply:
It's a bit redundant, since we can always just call `runtime.Version()` from the standard library and get the exact same data.